sdk/src/asset_handlers/sfnt_io.rs

```diff
@@ -1272,22 +1219,32 @@ where
         match chunk.chunk_type {
             // The table directory, other than the table records array will be
```

Comment:
We're now marking excluded chunks (and header/directory) as `Cai`, so I think this comment is out of place?